src/main.controller.js

```diff
@@ -1179,12 +1226,15 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 		 * Bootstraps plugin.
 		 */
 		function init () {
+			var isInboundConfig = true;
```

Comment:
I get why you're defining this here to be explicit about the purpose of the boolean, but maybe it should be done so next to the only spot in the logic where it is being used.

```
$scope.configs.forEach(config => {
  var isInboundconfig = true;
  escapeValues(config, isInboundConfig);
});
```

Reply:
That makes sense clarity-wise, but I was just trying to avoid re-declaring the same variable with the same value for each iteration. If you think it's harder to follow this way, I can update it.

Reply:
you can just put it right outside of the forEach. Thanks for humoring me on this minor detail.

Reply:
No problem. I'll test the IE fix tomorrow before the release.

Reply:
You're a capital fellow, Joe.